README.md

```diff
@@ -101,8 +101,8 @@ The connector uses the cross language [Spark SQL Data Source API](https://spark.
 ### Reading data from BigQuery
 
 ```
-df = spark.read
```

Comment:
As started, can you please fix all the code samples?

Reply:
updated.